modules/microshift-disconnected-host-con.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-disconnected-network-config.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-disconnected-host-preparation_{context}"]
+= Preparing networking for fully disconnected hosts
+
+Use the procedure that follows to start and run {microshift-short} clusters on devices running fully disconnected operating systems. A {microshift-short} host is considered fully disconnected if it has no addressable subnet.
+
+Typically this means that the device does not have an attached network interface controller (NIC) to provide a subnet. These steps can also be completed on a host with a NIC that is removed after setup. You can also automate these steps on a host that does not have a NIC by using the `%post` phase of a kickstart file.
+
+[IMPORTANT]
+====
+Configuring networking settings for disconnected environments is necessary because {microshift-short} requires a network device to support cluster communication. To meet this requirement, you must configure {microshift-short} networking settings to use the IP address of a system loopback device.
```

Comment:
```suggestion
Configuring networking settings for disconnected environments is necessary because {microshift-short} requires a network device to support cluster communication. To meet this requirement, you must configure {microshift-short} networking settings to use the IP address we will assign to the system loopback device.
```

The IP we're hinting at here doesn't exist until it's created during the setup (later called the "fake" IP addr).